test/e2e/db.js

```diff
@@ -57,6 +57,49 @@ const assertValidPrice = (t, p) => {
 	}
 }
 
+const assertValidTickets = (test, tickets) => {
+		test.ok(tickets);
+		for (let fare of tickets) {
+			if (fare.name !== undefined) {
+				test.equal(typeof fare.name, 'string');
+				test.ok(fare.name);
+			} else {
+				test.fail('Mandatory field "name" is missing');
```

Comment:
This case is covered by the `test.equal(typeof fare.name, 'string')` assertion above. Add an assertion message above, it will imply that the field is mandatory.

Same below.

Reply:
@derhuerst I adjusted the tests